av1_codec_registration.src.html

```diff
@@ -73,6 +73,45 @@
 contain a frame with a `frame_type` of `KEY_FRAME` as defined in Section
 6.8.2 of [[AV1]].
 
+VideoEncoderConfig extensions {#videoencoderconfig-extensions}
+==============================================================
+
+<pre class='idl'>
+<xmp>
+partial dictionary VideoEncoderConfig {
+  AV1EncoderConfig av1;
+};
+</xmp>
+</pre>
+
+<dl>
+  <dt><dfn dict-member for=VideoEncoderConfig>av1</dfn></dt>
+  <dd>
+    Contains codec specific configuration options for the AV1 codec.
+  </dd>
+</dl>
+
+AV1EncoderConfig {#av1-encoder-config}
+--------------------------------------
+<pre class='idl'>
+<xmp>
+dictionary AV1EncoderConfig {
+ boolean seqForceScreenContentTools = false
```

Comment:
Why the `seq` prefix - could this be `forceScreenContentTools`?

Reply:
Fixed.